todo_app/templates/index.html

```diff
@@ -1,18 +1,34 @@
 {% extends "layout.html" %}
-{% block title %}To-Do App{% endblock %}
-
+{% block title %}Trello Board{% endblock %}
 {% block content %}
   <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
+    <h1 class="display-5">To-Do Board</h1>
+    <p class="lead">Cards</p>
   </div>
-
   <div class="row">
     <div class="col-md-12">
-      <h2>Items</h2>
+      <h5>To-Do</h5>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for itm in items %} 
+          <li>{{ itm.id }} - {{itm.name}} - {{itm.desc}}</li> 
+        {% endfor %} 
       </ul>
+      <h5>Doing</h5>
+      <ul class="list-group mb-4">
+        {% for doitm in doingitems %} 
+          <li>{{ doitm.id }} - {{doitm.name}} - {{doitm.desc}}</li> 
+        {% endfor %}
+      </ul>
+      <h5>Done</h5>
+      <ul class="list-group mb-4">
+        {% for doneitm in doneitems %} 
+          <li>{{ doneitm.id }} - {{doneitm.name}} - {{doneitm.desc}}</li> 
+        {% endfor %}
+      </ul>
+      <form method="POST">
+        <input type="Submit" name="card" value="AddCard">
+        <input type="Submit" name="card" value="MoveCard">
+      </form>
```

Comment:
You could use links here

```suggestion
      <a href="{{ url_for('addCard') }}">Add card</a>
      <a href="{{ url_for('moveCard') }}">Move card</a>
```

that way your `/` index route doesn't need to accept POST requests.

If you want to style them like buttons, try using `class="btn btn-primary"` in the `a` tag.

Reply:
changed html code and used new style, thanks for the feedback